src/app/donation-start/donation-start-form/donation-start-form.component.ts

```diff
@@ -1204,30 +1199,6 @@ export class DonationStartFormComponent
     );
   }
 
-  captchaIdentityReturn(captchaResponse: string | null) {
```

Comment:
This fn just happens to be unused already as far as I can see, not directly related to the main validation change